solr/core/src/test/org/apache/solr/cloud/ReplicateFromLeaderTest.java

```diff
@@ -18,54 +18,94 @@
 package org.apache.solr.cloud;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
 
+import org.apache.solr.common.SolrException;
 import org.apache.solr.core.SolrConfig;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 public class ReplicateFromLeaderTest {
 
+  private String jettyTestMode;
+
+  @Before
+  public void setUp() {
+    jettyTestMode = System.getProperty("jetty.testMode");
+    System.clearProperty("jetty.testMode");
+  }
+
+  @After
+  public void tearDown() {
+    if (jettyTestMode != null) {
+      System.setProperty("jetty.testMode", jettyTestMode);
+    }
+  }
+
```

Comment:
Would retaining existing behaviour -- https://github.com/apache/solr/pull/2316/files#r1755039181 -- mean that we don't have to make this test change here?

Reply:
Yes, this is due to `jetty.testMode` being set during unit tests by default :-/